test/test_other.py

```diff
@@ -9730,6 +9730,13 @@ def test_noderawfs_access_abspath(self):
     self.run_process([EMCC, 'access.c', '-sNODERAWFS'])
     self.run_js('a.out.js', args=[os.path.abspath('foo')])
 
+  def test_noderawfs_readfile_prerun(self):
+    create_file('foo', 'bar')
+    create_file('pre.js', f'''
+      Module.preRun = () => console.log(FS.readFile('{os.path.abspath('foo')}', {{ encoding: 'utf8' }}));
```

Comment:
Do you need `{ encoding: 'utf8' }` here?

Also can you just use `FS.readFile('foo')`?

Reply:
`{ encoding: 'utf8' }` is needed, otherwise it would print: `Uint8Array(3) [ 98, 97, 114 ]`.

`FS.readFile('foo')` is done with commit 57b3cc096e0a13f0edd779a7fcf12c8a8629d1c4,